api/app/src/main/resources/errorBundle.properties

```diff
@@ -14,4 +14,5 @@ invalidPermissionsProvided=Invalid permissions provided
 userAlreadyExists=User already exists
 userNotFound=User not found
 roleAlreadyExists=Role already exists
-adminRoleNotFound=Admin role not found, contact your administrator
\ No newline at end of file
+roleNotFound=Role not found
+adminRoleNotFound=Admin role not found, contact your administrator
```

Comment:
It might be tidier to keep these keys arranged in alphabetical order... or arranged into meaningful categories.

Reply:
yup agree will put as part of last PR (user delete) the subsequent ones add more